internal/nix/nix.go

```diff
@@ -412,3 +413,21 @@ func parseInsecurePackagesFromExitError(errorMsg string) []string {
 
 	return insecurePackages
 }
+
+func RestartDaemon(ctx context.Context) error {
+	if runtime.GOOS != "darwin" {
+		return nil
```

Comment:
Should we print a warning telling user to restart? (only if non-darwin multi-user install)